src/main/java/org/shanerx/tradeshop/shop/ShopChest.java

```diff
@@ -50,7 +52,7 @@
 
 public class ShopChest extends Utils {
 
-	private final transient static TradeShop plugin = (TradeShop) Bukkit.getPluginManager().getPlugin("TradeShop");
+	private final static TradeShop plugin = (TradeShop) Bukkit.getPluginManager().getPlugin("TradeShop");
```

Comment:
Transient not needed for static fields apparently and the newer version of Idea catches this.

Reply:
I was wondering what this is about